packages/manager/apps/catalog/src/api/GET/aapi/service.ts

```diff
@@ -18,39 +17,12 @@ export type Product = {
   category: string;
 };
 
-export type CatalogData = {
-  data: Product[];
-};
-
-export type ResponseData<T> = {
-  status: number;
-  data: T;
-  code: string;
-  response?: {
-    status: number;
-    data: { message: string };
-  };
-};
-
 /**
  * Manager-catalog endpoints : Get manager-catalog
```

Comment:
It's a 2API endpoint and not Manager endpoint.
```suggestion
 * 2API-catalog endpoints : Get catalog
```

Reply:
done